src/cbc_sdk/platform/alerts.py

```diff
@@ -252,6 +252,29 @@ def update_threat(self, remediation=None, comment=None):
         """
         return self._update_threat_workflow_status("OPEN", remediation, comment)
 
+    @staticmethod
+    def search_suggestions(cb, query):
```

Comment:
Thought it is a tautology for the name to be `alerts_search_suggestions`, but if you think it is better to use `alerts_search_suggestions` I will change it.

Reply:
`search_suggestions` makes sense as it will be `Alert.search_suggestions`